cfgrib/dataset.py

```diff
@@ -221,6 +221,12 @@
         'standard_name': 'forecast_reference_time',
         'long_name': 'initial time of forecast',
     },
+    'indexing_time': {
+        'units': 'seconds since 1970-01-01T00:00:00',
+        'calendar': 'proleptic_gregorian',
+        'standard_name': 'time',
```

Comment:
It the standard name `time` like `valid_time` or `forecast_reference_time`?

Reply:
It's a forecast_reference_time. I have fixed standard_name and long name.